crawler/crawler.py

```diff
@@ -287,28 +288,50 @@ def crawler_worker(process_id, logfile, params):
 
     crawlutils.snapshot(**params)
 
+def docker_monitor(process_id, logfile, syncQ):
+    setup_logger('docker-monitor', logfile, process_id)
```

Comment:
Can you log to crawler.log instead please? notice that crawler-[0-9].log is for the workers.